integration-tests/helpers.js

```diff
@@ -188,6 +188,14 @@ function spawnProc (filename, options = {}, stdioHandler) {
   })
 }
 
+async function createSandboxForNext (dependencies = [],
```

Comment:
How about instead of creating a new `createSandboxForNext` function that is specific just to next, we instead modify the `createSandbox` to add a generically useful follow-up command?

Something like this:

```js
async function createSandbox (
  dependencies = [],
  isGitRepo = false,
  integrationTestsPaths = ['./integration-tests/*'],
  followUpCommand
) {
  // ...
  if (followUpCommand) {
    await exec(followUpCommand, { cwd: sandbox.folder, env: restOfEnv })
  }

  return ...
}
```

Then in the calling case you'd do something like this:

```js
    // old
    sandbox = await createSandboxForNext(['next', 'react', 'react-dom'],
      false, ['./packages/datadog-plugin-next/test/integration-test/*'])

    // new
    sandbox = await createSandbox(['next', 'react', 'react-dom'],
      false, ['./packages/datadog-plugin-next/test/integration-test/*'],
      'yarn exec next build')
```

This is because the helpers.js file is more universal and we don't want to add super specific one off cases to it like for Next support.

Reply:
I agree with this - maybe even making it more general could be beneficial? Something like having `followUpCommand` being a function that's executed instead, in the case where something has more than one follow up command (I'm not sure if there are many ESM tests left, but maybe for future use).